### ISSUE TITLE
mail address cannot login Mailcow UI. SOGo login works.

### ISSUE THREAD

**Issue by oehTie** [mailcow/mailcow-dockerized#867]
Hello,

I Installed Mailcow some time ago but didn't really use it. Now i've started to use it, updated it to latest stable branch, but i've encountered a problem with logging in into the Mailcow UI to edit account specific settings.

I know the password is correct, sometimes changing it via the admin account can help fixing the problem but then it might start again later. Logging in into SOGo for example works without problem. Also sending or receiving mail works without problem.

Any clue where to look for the error? I can't really find log files in /var/log that tell me what is wrong...

thanks in advance.

Theo

**Comment by mritzmann** [comment]
I had the same problem in December.

> sometimes changing it via the admin account can help fixing the problem

Same for me. After set the same passwort again, fixed the problem for me.

**Comment by oehTie** [comment]
i tried that here too and yes i can log in again. But i have no clue if after some time i get locked out again. And no clue why... So i would like to search the logs what went wrong, only no clue where to find them.
 
resetting the password is only a workaround

**Comment by hachre** [comment]
I'm not really understanding the problem. How can you have problems logging in but then change the password to fix it? How do you change the password if you can't log in?

The logs can be accessed through customizing the following example:
```
cd /opt/mailcow-dockerized
docker-compose logs --tail=1000 dovecot-mailcow
```

You can use `docker-compose ps` to find the names of the containers that you can use. `--tail=1000` allows you to reduce the output to the last 1000 lines for example.

**Comment by oehTie** [comment]
Thanks hachre, that shows logs for the various docker containers. 

I currently do not have an account with the issue anymore, because i used the workaround i described above... as soon as i can reproduce the issue i'll post logs here.

**Comment by mritzmann** [comment]
> How do you change the password if you can't log in?

With admin Account. Only the login to mailcow with an mailbox account is affected.

**Comment by hachre** [comment]
Ah, I see - weird. I haven't had this problem myself yet.

**Comment by andryyy** [comment]
Hm, as soon as anyone hits this error, please contact me at info@servercow.de, I will check and fix it.

**Comment by classner** [comment]
Hi!

Not sure whether this is the same problem, but it seems similar to me. I upgraded mailcow today. After that, I created a new user. For this user sending and receiving email works all fine with the credentials set (!!), but login in the mailcow UI or the SOGO connector both doesn't work.

Best,
Christoph

Edit: is solved and was unrelated.

**Comment by stevesbrain** [comment]
@andryyy I will let you know - I've had this error before. Wasn't sure if it was to do with certain symbols in the password? Definitely had the error on at least 3 separate occasions though :)

**Comment by andryyy** [comment]
Damn, I was stupid for using ltrim.

**Comment by amanjuman** [comment]
I'm having the same issue.
Created 3 users from UI and can't login webmail and it's telling invalid password.

**Comment by mritzmann** [comment]
> I'm having the same issue.

@amanjuman The problem mentioned in this issue was fixed in 2018 by ea6c6a5e84cb261593b29a3a4d4b1498b6879b2b. This ticket has been closed. I recommend creating a new issue for your problem.
